src/main/java/edu/kis/powp/jobs2d/drivers/RealTimeDrawingDriver.java

```diff
@@ -0,0 +1,50 @@
+package edu.kis.powp.jobs2d.drivers;
+
+import edu.kis.legacy.drawer.panel.DrawPanelController;
+import edu.kis.legacy.drawer.shape.ILine;
+import edu.kis.powp.jobs2d.drivers.adapter.LineDriverAdapter;
+
+import javax.swing.*;
+import java.util.LinkedList;
+import java.util.Queue;
+
+public class RealTimeDrawingDriver extends LineDriverAdapter {
```

Comment:
It would be best to provide such ability to all possible drivers. However this would require using Decorator approach!